test.js

```diff
@@ -3,11 +3,13 @@ import childProcess from 'child_process';
 import test from 'ava';
 import noopProcess from 'noop-process';
 import processExists from 'process-exists';
+import delay from 'delay';
 import m from './';
 
 test('pid', async t => {
 	const pid = await noopProcess();
 	await m(pid, {force: true});
+	await delay(10);
```

Comment:
Can you make the number a top-level constant with a descriptive name and a comment on why it's needed?